ospfd/ospfd.c

```diff
@@ -2177,10 +2177,26 @@ static int ospf_vrf_disable(struct vrf *vrf)
 	return 0;
 }
 
+static int ospf_vrf_config_write(struct vty *vty)
```

Comment:
This code breaks some assumptions about how we expect data to be outputed for cli actions, namely we don't display a mode if there is no data in the sub-mode.  If you must have a vrf data then cause the `router ospf vrf XX` line to auto-create the vrf if it does not exist instead.

Reply:
ok. I understand your point.
I have no solutio for now, except the following:
For instance, if I want to permit vrf creation on ospfd, I should modify ospf_vrf_new() so as to create a specific context. 
Then if that context exists, the vrf will be displayed on the running-config, ( then on startup configuration too).

ok for you?